stl/inc/yvals.h

```diff
@@ -7,6 +7,12 @@
 #ifndef _YVALS
 #define _YVALS
 #include <yvals_core.h>
+
+#ifdef _ENFORCE_ONLY_CORE_HEADERS
+static_assert(false, "error STL2001: You tried to include a non-core C++ standard header file. "

```

Comment:
What about "Tried to include a non-core C++ Standard Library header file with _ENFORCE_ONLY_CORE_HEADERS defined". Sounds more errory to me